test/zip_view_test.cpp

```diff
@@ -152,61 +152,63 @@ TEST(ZipView, iterator_modify) {
 }
 // Two following tests disabled because of unresolved issues with
 // conversion from pair to tuple as part of copy and copy_if
+#ifndef _CLANG_
```

Comment:
It actually works fine with clang on Linux, do you want to disable it OSX-specific?